euth_wagtail/assets/scss/components/_dashboard.scss

```diff
@@ -139,6 +139,14 @@ h1.dashboard-title {
     background-color: #fff;
     text-align: center;
     padding: 1em;
+
+    > button {
```

Comment:
I see. Using `.btn` would have side effects for the `choose template` button. And I didn't like the idea to have an extra css class just for the `learn more` button.